packages/mco/components/data-policies/drpolicy-list-page/drpolicy-list-page.tsx

```diff
@@ -252,51 +253,59 @@ export const DRPolicyListPage: React.FC = () => {
       cluster: HUB_CLUSTER_NAME,
     });
 
+  const canCreateDRPolicy = useAccessReviewAllowed({
```

Comment:
this returns loading state as well... we can pass it in `DRPolicyList`...
```suggestion
  const [canCreateDRPolicy, canCreateDRPolicyLoading] = useAccessReview({
```